ml-agents/mlagents/trainers/demo_loader.py

```diff
@@ -31,7 +31,7 @@ def make_demo_buffer(
     for idx, experience in enumerate(pair_infos):
         if idx > len(pair_infos) - 2:
             break
-        current_pair_info = pair_infos[idx]
+        current_pair_info = experience
```

Comment:
minor, but this should probably just be named `current_pair_info` in the `for` statement

Reply:
Good call, that's better.